.changeset/spicy-plums-switch.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/itwinui-react": patch
+---
+
+Bumped the `@floating-ui/react` dependency from `^0.26.3` to `^0.26.10`.
```

Comment:
Just wanted to confirm before merging, is the new changeset alright? Also, should this be a `minor` changeset instead since we suspect that floating-ui's patch releases might contain breaking behavioral changes?

Reply:
something like this?

```suggestion
---
"@itwin/itwinui-react": minor
---

Bumped the minimum required version of `@floating-ui/react` from `^0.26.3` to `^0.26.10`. (Make sure to also update transitive deps!)
```